aws-datastore/src/main/java/com/amplifyframework/datastore/AWSDataStorePlugin.java

```diff
@@ -191,6 +191,18 @@ public CategoryType getCategoryType() {
             new ResultConversionListener<>(saveItemListener, this::toDataStoreItemChange));
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public <T extends Model> void save(
```

Comment:
Should the version at L187 just call this one, now?

Reply:
Now that I've made `DataStoreCategoryBehavior`'s predicates into `@NonNull`, I can't call the it with `null` predicate anymore, so I'll just leave it directly calling the method from storage adapter.